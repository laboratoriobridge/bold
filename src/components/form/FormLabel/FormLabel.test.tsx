import React from 'react'
import { render } from 'react-testing-library'

import { FormLabel } from './'

it('should render correctly', () => {
  expect(render(<FormLabel label='Label' />).container).toMatchSnapshot()
  expect(render(<FormLabel label='Label' htmlFor='test' required />).container).toMatchSnapshot()
})

it('should accept the style prop', () => {
  expect(render(<FormLabel label='Label' style={{ color: 'blue' }} />).container).toMatchSnapshot()
})
