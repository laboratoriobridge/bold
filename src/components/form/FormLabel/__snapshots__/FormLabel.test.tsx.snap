// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should accept the style prop 1`] = `
.emotion-0 {
  font-weight: bold;
  color: blue;
}

<div>
  <label
    class="emotion-0"
  >
    Label
  </label>
</div>
`;

exports[`should render correctly 1`] = `
.emotion-0 {
  font-weight: bold;
}

<div>
  <label
    class="emotion-0"
  >
    Label
  </label>
</div>
`;

exports[`should render correctly 2`] = `
.emotion-1 {
  font-weight: bold;
}

.emotion-0 {
  color: #D83A2E;
  margin-left: 0.25rem;
}

<div>
  <label
    class="emotion-1"
    for="test"
  >
    Label
    <span
      class="emotion-0"
      title="Campo obrigatÃ³rio"
    >
      *
    </span>
  </label>
</div>
`;
